src/unix/linux_like/linux/mod.rs

```diff
@@ -504,6 +504,11 @@ s! {
         pub ee_info: u32,
         pub ee_data: u32,
     }
+
+    pub struct f_owner_ex {
+        pub type_: ::c_int,
```

Comment:
We should add `f_owner_ex` here:

https://github.com/rust-lang/libc/blob/0d374002b8d4972e97e04b0a7fb1a210e22368ed/libc-test/build.rs#L2513-L2522